dockers/docker-lldp-sv2/start.sh

```diff
@@ -2,6 +2,11 @@
 
 sonic-cfggen -d -t /usr/share/sonic/templates/lldpd.conf.j2 > /etc/lldpd.conf
 
+# update the hostname. Not using device metadata as in multi-npu platform each namespace
+# has its own device metadata hostname. Using the hostname set by docker using host enviroment
+
+sed -e "s/\${hostname}/$(hostname)/" -i /etc/lldpd.conf 
```

Comment:
Where hostname is defined?

Reply:
@pavel-shirshov this is linux hostnaname based of /etc/host

Reply:
@pavel-shirshov this change is no more applicable and we will continue to use Device Metadata hostname.